File: Webserver/backend/Utilities/SplitPheno.py
import rpy2.robjects as robjects
from rpy2.robjects.packages import importr

GEOquery = importr("GEOquery")
Biobase = importr("Biobase")

print(GEOquery)
data = GEOquery.getGE0("GSE19711", filename="/home/data/Download/GSE19711_series matrix,txt.gz",destdir = '.',
                       AnnotGPL=False, getGPL=False)

pheno = Biobase.pData(data)
beta = Biobase.exprs(data)

File: Webserver/backend/R/Splitpheno.R
library(GEOquery)

setwd('/home/data/Download/')
data <- getGEO("GSE41169", filename="GSE41169_series_matrix.txt.gz", destdir = '', AnnotGPL=F, getGPL = F)


pheno <- pData(data)

beta <- exprs(data)

colnames(beta)
pheno$geo_accession


write.csv(pheno,"/home/data/Raw/Meta/GSE41169_pheno.csv",row.names=FALSE)
write.csv(beta,"/home/data/Raw/Beta/GSE41169_beta.csv",row.names=FALSE)


data <- getGEO('GSE41169', destdir = '/home/data/Download/', AnnotGPL=F, getGPL = F)

beta <- exprs(data[[1]])

pheno <- pData(data[[1]])

